frontend/.gitignore

```diff
@@ -34,4 +34,7 @@ yarn-error.log*
 .vercel
 
 test/cypress/videos
-test/cypress/screenshots
\ No newline at end of file
+test/cypress/screenshots
+
+# Sentry
+.sentryclirc
```

Comment:
FYI We're not using this file (this line was auto-generated), but it can optionally contain the SENTRY_AUTH_TOKEN which is why it's excluded from VCS.